iOS/SideDish-team24/SideDish-team24/Model/Dish.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+class Dish {
+    private let id: String
+    private let name: String
+    private let price : Int
+    private let stock : Int
+    private let type : String
+    private let image: URL
+    private let description: String
+    
+    private let discountPrice : Int?
+    private let discountPolicy : String?
+    
+    init(id: String, name: String, price: Int, stock: Int, type: String, image: URL, description: String) {
+        self.id = id
+        self.name = name
+        self.price = price
+        self.stock = stock
+        self.type = type
+        self.image = image
+        self.description = description
+        self.discountPrice = nil
+        self.discountPolicy = nil
+    }
+    
+    init(id: String, name: String, price: Int, stock: Int, type: String, image: URL, description: String, discountPrice: Int, discountPolicy: String) {
```

Comment:
초기값을 nil 로 줘서 init을 하나로 만들 수 있을 것 같은데, 이렇게 하신 이유가 있으실까요? 🤔
```suggestion
    init(id: String, name: String, price: Int, stock: Int, type: String, image: URL, description: String, discountPrice: Int? = nil, discountPolicy: String? = nil) {
```

Reply:
생각치 못했습니다! 수정하겠습니다 🤗